backend/src/main/java/ai/verta/modeldb/project/ProjectServiceImpl.java

```diff
@@ -720,7 +721,9 @@ public void getProjects(
       saveAuditLog(
           Optional.of(userInfo),
           ModelDBServiceActions.READ,
-          ModelDBConstants.EMPTY_STRING,
+          projects.stream()
+              .map(Project::getId)
+              .collect(Collectors.joining(ModelDBConstants.COMMA_DELIMITER)),
```

Comment:
Can you add a TODO of "don't save every single ID on fetch"? We should just move on from that right now, but this will become massive later on with us listing every single id we touch.

Reply:
Added TODO